scripts/dp/lib/create-order.js

```diff
@@ -85,16 +85,10 @@ function createOrder(augur, marketId, outcome, numOutcomes, maxPrice, minPrice,
       _tradeGroupId: tradeGroupId,
       doNotCreateOrders: false,
       onSent: function (res) {
-        if (debugOptions.cannedMarkets) console.log(chalk.green.dim("placeTrade sent:"), res);
-      },
-      onSuccess: function (tradeOnChainAmountRemaining) {
-        if (debugOptions.cannedMarkets) console.log(chalk.green.dim("placeTrade success:"), tradeOnChainAmountRemaining);
-        callback(null);
-      },
-      onFailed: function (err) {
-        if (debugOptions.cannedMarkets) console.error(chalk.red.bold("placeTrade failed:"), err);
-        callback(err);
+        if (debugOptions.cannedMarkets) console.log(chalk.green.dim("placeTrade sent:"), res.hash);
       },
+      onSuccess: function () { callback(null); },
+      onFailed: callback,
```

Comment:
It seems like this should be passing an error back to the callback function. Why was this removed?

Reply:
It still does.